src/components/ListItem.jsx

```diff
@@ -27,7 +28,7 @@ export function ListItem({ name, item, listToken, urgencyCategory }) {
 	};
 
 	return (
-		<div>
+		<div className="ListItemContainer">
```

Comment:
This is working great, so no need to change. For more MUI learning [you might checkout the <Box/> component](https://mui.com/material-ui/react-box/); it is the general wrapping component for CSS styling.

Reply:
Big ➕ for @wheninseattle suggestion. One of the first things I noticed as well but can be done as a refactor to keep elements in MUI as much as possible